projects/plugins/crm/src/automation/commons/triggers/companies/class-company-created.php

```diff
@@ -75,11 +75,10 @@ public static function get_data_type(): string {
 	 * Listen to the desired event.
 	 *
 	 * @since $$next-version$$
+	 *
+	 * @return void
 	 */
-	protected function listen_to_event() {
-		add_action(
-			'jpcrm_company_created',
-			array( $this, 'execute_workflow' )
-		);
+	protected function listen_to_event(): void {
+		$this->listen_to_wp_action( 'jpcrm_company_created' );
```

Comment:
Use the implemented shortcut for WP hooks. But the trigger implementation could listen to any other system or perform some actions here if they want, for instance, the `WP_User_Created` trigger doesn't use the shortcut because it needs to do some actions.